progC/main.c

```diff
@@ -0,0 +1,37 @@
+
+
+
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(int argc, char *argv[])
+{   
+    FILE * file;
+    char* my_var  = getenv("MY_VAR");
+    if(!my_var){
+       return 123;
+    }
+    else 
+    {
+        if (argc > 1)
+        {
+            file = fopen("env.txt", "a");
```

Comment:
provide an ability to write to file from argv 
example :
./main -0 file.txt -> write to file.txt
./main -0 mood.txt -> write to mood.txt